packages/api/src/controllers/playback.ts

```diff
@@ -89,16 +89,13 @@ const getAssetPlaybackUrl = async (
   }
   const playbackUrl = assetPlaybackUrl(config, ingest, asset, os);
   const staticFilesPlaybackInfo = getStaticPlaybackInfo(asset, os);
-  const inExperiment = await isExperimentSubject(
-    "lit-signing-condition",
-    asset.userId
-  );
+
   return !playbackUrl
     ? null
     : {
         staticFilesPlaybackInfo,
         playbackUrl,
-        playbackPolicy: inExperiment ? asset.playbackPolicy : null,
+        playbackPolicy: asset.playbackPolicy || null,
```

Comment:
This `||` is unnecessary. IMO it's better to not include the field if it doesn't exist instead of specifying it as `null` (my bad on including it as `null` in the first place)
```suggestion
        playbackPolicy: asset.playbackPolicy,
```